### PR TITLE
Bug 1219268 - test_browser_play_video.py fails on smoke test due to wait_for_page_to_load() failure

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6fb34a933ae42a62c0b44029d7ab0c2d5668b653

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7e462129b5de63d9e4524af7a2b4b2e49390a96f

File: tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py [hunk 1]
```diff
@@ -39,8 +39,10 @@ def switch_to_chrome(self):
     def is_page_loading(self):
         return self._root_element.get_attribute('loading-state') == "true"
 
+    def wait_for_page_to_start_load(self):
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py]: I'm not sure, do we say start load or start loading?

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py]: loading, I'll change it before I merge.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=feb26d2c3cbf5ea0b578bad08272ab98c115bae7

**Pull request merged**